components/eamxx/tests/python/pyp3/p3_standalone_py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+import sys
+
+# Add path to scream libs
+sys.path.append('@SCREAM_BASE_DIR@/scripts')
+
+# Add path to pyeamxx libs
+sys.path.append('@CMAKE_BINARY_DIR@/src/share/python')
+sys.path.append('@CMAKE_BINARY_DIR@/src/physics/p3/python')
+
+# Without these, and manual init/finalize, on my laptop I get
```

Comment:
If/when we move MPI/eamxx init/final in the module internals, this may become moot.

Reply:
Btw, I think [this approach](https://github.com/sandialabs/Albany/blob/master/pyAlbany/python/__init__.py) is quite neat. The temporary is created at module level, so it is destroyed when the module is unloaded, and its destructor will take care of finalizing kokkos. We can do something similar with MPI/eamxx init/final.